dali/operators/image/color/color_twist.cc

```diff
@@ -24,51 +24,61 @@ using TheKernel = kernels::LinearTransformationCpu<Out, In, 3, 3, 3>;
 }  // namespace
 
 DALI_SCHEMA(Hsv)
-    .DocStr(R"code(This operator performs HSV manipulation.
-To change hue, saturation and/or value of the image, pass corresponding coefficients.
-Keep in mind, that `hue` has additive delta argument,
-while for `saturation` and `value` they are multiplicative.
+    .DocStr(R"code(This operator performs an HSV manipulation.
 
-This operator accepts RGB color space as an input.
+To change the hue, the saturation, and/or the value of the image, pass the corresponding
+coefficients. Remember that the ``hue`` has an additive delta argument,
+while for ``saturation`` and value, the arguments are multiplicative.
 
-For performance reasons, the operation is approximated by a linear transform in RGB space.
+This operator accepts the RGB color space as an input.
+
+For performance reasons, the operation is approximated by a linear transform in the RGB space.
 The color vector is projected along the neutral (gray) axis,
-rotated (according to hue delta) and scaled according to value and saturation multiplers,
-and then restored to original color space.
-)code")
+is rotated based on the hue delta, is scaled based on the value and saturation multipliers,
+and is restored to the original color space.)code")
     .NumInput(1)
     .NumOutput(1)
     .AddOptionalArg(color::kHue,
-                    R"code(Set additive change of hue. 0 denotes no-op)code",
+                    R"code(Sets the additive change of the hue.
+
+A value of 0 denotes no-op.)code",
```

Comment:
```suggestion
                    R"code(Hue delta, in degrees; 0 is a no-op.
                    
The hue component can be interpreted as an angle and values outside 0-360 range wrap around, as they would in case of rotation.)code",
```

Reply:
Done